pages/payruns/[guid]/index.js

```diff
@@ -74,6 +75,9 @@ const SinglePayRun = () => {
     update();
     updateInsight();
   };
+
+  const isLoading = invoiceLoading || insightsIsLoading || loading;
```

Comment:
nope we dont want this logic being passed in

Reply:
removed loading